CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Internal
 
+- Enabling support for the new active LTS NodeJS version 14. NodeJS 10 eol will happen on 2021-04-30 and Volto will update accordingly. More information on https://nodejs.org/en/about/releases @sneridagh
```

Comment:
@sneridagh I think I'd make this a feature release. We are actually adding a new feature which is that we commit ourselves to support a new node version (and test it properly). Technically we are not doing anything here. Though, it is important for outside communication IMHO and should not go into something subtle as "internal".